MitigationApi.yml

```diff
@@ -598,6 +598,18 @@ components:
       - DENIED
       default: RECOMMENDED
       
+    Resolution:
+      type: string
+      description: "
+        This is a description of 
+        MANUAL describes 

```

Comment:
This description could be.....better. Looks like it was a placeholder that never quite got filled out.